### ISSUE TITLE
Demo

### ISSUE THREAD

**Issue by iehdk** [serut/meteor-coverage#76]
Is there a demo of the result? A nice html page?

**Comment by serut** [comment]
Hello ! 

There is still issues with the 3.0 release, I'm still working on it : 
- meteortesting:mocha runner conflict with my last version ([very minor issue](https://github.com/meteortesting/meteor-mocha/blob/master/package/package.js#L14))
- the HTML report is not working well (not that complicated too)

For open source apps, you can get the following result : https://app.codacy.com/project/l-mieulet/meteor-coverage/dashboard

For private sources, you can generate an HTML report (that is not working right now) that tracks for each file the coverage.

**Comment by iehdk** [comment]
Yup, I got it running. I think that very nice screenshot should be added to the README.md ;o)
